tests/robot-tests/tests/admin_and_public/bau/publish_data.robot

```diff
@@ -457,19 +452,28 @@ Select subject "${SUBJECT_2_NAME}" in table tool
     user waits until table tool wizard step is available    2    Choose a subject
     user waits until page contains    ${SUBJECT_2_NAME}
     user clicks radio    ${SUBJECT_2_NAME}
-    user clicks element    id:publicationSubjectForm-submit
+    user waits until button is enabled    Next step    %{WAIT_SMALL}
+    user clicks button    Next step
     user waits until table tool wizard step is available    3    Choose locations
     user checks previous table tool step contains    2    Subject    ${SUBJECT_2_NAME}
 
 Select locations in table tool
-    user opens details dropdown    Local Authority
+    # LH: Not using the 'user opens details dropdown' keyword here as
+    # it opens the 'Local Authority District' dropdown instead
+    # due to using 'contains'
+    # TODO: LH - fix this
```

Comment:
I'll still approve, but this kind of info is best stored in a ticket - and should probably be more specific about what you intend to solve. Otherwise you might come back to this and not remember what the issue is.